api/src/main/kotlin/edu/wgu/osmt/security/OAuthHelper.kt

```diff
@@ -13,12 +15,19 @@ import org.springframework.stereotype.Component
 @Component
 class OAuthHelper {
 
+    @Autowired
+    lateinit var appConfig: AppConfig
+
     fun readableUsername(user: OAuth2User?, default: String = UNAUTHENTICATED_USERNAME): String {
         return user?.name ?: default
     }
 
     fun readableUsername(jwt: Jwt?, default: String = UNAUTHENTICATED_USERNAME): String {
```

Comment:
Let's please case this properly - readableUserName

Reply:
nice Catch!